jina/types/document/__init__.py

```diff
@@ -402,7 +403,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
-        self.update_id()
+        pass
```

Comment:
So is the Document supposed to be used as a context manager now or not?

Reply:
In my opinion, you shouldn't for now. But since it is advertised in the Native Data Types blog article and is also already used all over the place, we choose to keep the possibility.